### PR TITLE
SAK-49881 Wiki Search results do not follow dark mode

### PR DESCRIPTION
**Pull request by jonespm**
This may have fixed some other issues with Wiki as I updated everything I saw on this search page w.r.t. dark mode.

Some of these components may be better being converted to more standard ones but there's custom logic in this tool for the buttons so that would take more effort than just fixing the dark mode theming.

### CODE REVIEW

File: library/src/skins/default/src/sass/modules/tool/wiki/_wiki.scss [hunk 1]
```diff
@@ -490,7 +492,11 @@
       color: var(--sakai-text-color-2);
   }
 
-  .lastmodified, .groupsforthispage { font-size: smaller; text-align: right; color: #555;padding:.5em 0}
+  .lastmodified, .groupsforthispage { 
+      font-size: smaller; 
+      text-align: right; 
+      padding:.5em 0
```
Comment (kunaljaykam) [inline library/src/skins/default/src/sass/modules/tool/wiki/_wiki.scss:498]:
```suggestion
      padding:.5em 0;
```

File: library/src/skins/default/src/sass/modules/tool/wiki/_wiki.scss [hunk 2]
```diff
@@ -261,12 +261,12 @@
       border:1px solid #ccc;
       list-style-position: inside;
       margin:0;padding:0 1em 0 0 ;
-      background:#ffe;
+      background:var(--sakai-background-color-2)
```
Comment (kunaljaykam) [inline library/src/skins/default/src/sass/modules/tool/wiki/_wiki.scss:264]:
```suggestion
      background:var(--sakai-background-color-2);
```

File: library/src/skins/default/src/sass/modules/tool/wiki/_wiki.scss [hunk 3]
```diff
@@ -261,12 +261,12 @@
       border:1px solid #ccc;
       list-style-position: inside;
       margin:0;padding:0 1em 0 0 ;
```
Comment (kunaljaykam) [inline library/src/skins/default/src/sass/modules/tool/wiki/_wiki.scss:263]:
```suggestion
      margin: 0;
      padding: 0 1em 0 0;
```

File: library/src/skins/default/src/sass/modules/tool/wiki/_wiki.scss [hunk 4]
```diff
@@ -490,7 +492,11 @@
...
   }
 
-  .lastmodified, .groupsforthispage { font-size: smaller; text-align: right; color: #555;padding:.5em 0}
+  .lastmodified, .groupsforthispage { 
+      font-size: smaller; 
+      text-align: right; 
+      padding:.5em 0
+   }
   
   .bold-text { font-weight: bold; color:#000 }
```
Comment (kunaljaykam) [inline library/src/skins/default/src/sass/modules/tool/wiki/_wiki.scss:501]: I think we can remove this line. And Use B-5 classes instead.

File: library/src/skins/default/src/sass/modules/tool/wiki/_wiki.scss [hunk 5]
```diff
@@ -790,7 +800,7 @@
    * Feel free to use this on your websites, but please leave this message in the fies
    * http://www.silverstripe.com/blog
    */
-  ul.tree{ width: auto; padding-left: 0px; margin-left: 0px;border:1px solid #ccc;padding:5px;background:#ffe;}
+  ul.tree{ width: auto; padding-left: 0px; margin-left: 0px;border:1px solid #ccc;padding:5px;}
```
Comment (kunaljaykam) [inline library/src/skins/default/src/sass/modules/tool/wiki/_wiki.scss:803]:
```suggestion
  ul.tree{ width: auto; padding-left: 0px; margin-left: 0px; border: 1px solid #ccc; padding: 5px;}
```

File: library/src/skins/default/src/sass/modules/tool/wiki/_wiki.scss [hunk 6]
```diff
@@ -261,12 +261,12 @@
       border:1px solid #ccc;
```
Comment (kunaljaykam) [inline library/src/skins/default/src/sass/modules/tool/wiki/_wiki.scss:261]:
```suggestion
      border: 1px solid #ccc;
```

**Review by kunaljaykam — COMMENTED** [review]
I've noticed a few inconsistencies regarding spacing and the semicolons in ` _wiki.scss`. While you've fixed some of these issues, There are still some left. It's surprising that these didn't caused any errors, Could you take another look? Thanks

File: library/src/skins/default/src/sass/modules/tool/wiki/_wiki.scss (hunk 4 above)
Comment (ottenhoff) [reply library/src/skins/default/src/sass/modules/tool/wiki/_wiki.scss:502]: rwiki-tool/tool/src/webapp/WEB-INF/vm/view.vm line 91

File: library/src/skins/default/src/sass/modules/tool/wiki/_wiki.scss [hunk 7]
```diff
@@ -490,7 +493,11 @@
...
   }
 
-  .lastmodified, .groupsforthispage { font-size: smaller; text-align: right; color: #555;padding:.5em 0}
+  .lastmodified, .groupsforthispage { 
+      font-size: smaller; 
+      text-align: right; 
+      padding:.5em 0;
+   }
   
   .bold-text { font-weight: bold; color:#000 }
```
Comment (kunaljaykam) [inline library/src/skins/default/src/sass/modules/tool/wiki/_wiki.scss:502]: @ottenhoff Perhaps we can add `fw-bold` class to view.vm line 91. I don't see this style used anywhere else. then we can remove it. I can do a pr in Matt's branch for this or you can just edit it , if you have edit privileges.
```suggestion
```

File: library/src/skins/default/src/sass/modules/tool/wiki/_wiki.scss (hunk 4 above)
Comment (ottenhoff) [reply library/src/skins/default/src/sass/modules/tool/wiki/_wiki.scss]: done

**Review by kunaljaykam — APPROVED** [review]

**Review by ern — APPROVED** [review]

**Pull request merged**